hc/api/forms.py

```diff
@@ -0,0 +1,5 @@
+from django import forms
+
+
+class TeamMemberForm(forms.Form):
+    member_id = forms.IntegerField(required=True)
```

Comment:
Add new line at the end of this file. Get an extension for this for your editor.